src/server/scripts/Outland/zone_hellfire_peninsula.cpp

```diff
@@ -420,10 +420,462 @@ class npc_fel_guard_hound : public CreatureScript
+
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 8000);
+                                step++;
+                                break;
+                            case 13:
+                                Talk(SAY_BARADA_5);
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 14:
+                                if (jules)
+                                    jules->AI()->Talk(SAY_JULES_4);
+
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 15:
+                                Talk(SAY_BARADA_6);
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 16:
+                                if (jules)
+                                    jules->AI()->Talk(SAY_JULES_5);
+
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 17:
+                                Talk(SAY_BARADA_7);
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 18:
+                                if (jules)
+                                    jules->AI()->Talk(SAY_JULES_3);
+
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 19:
+                                Talk(SAY_BARADA_7);
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 20:
+                                if (jules)
+                                {
+                                    jules->GetAI()->DoAction(ACTION_JULES_MOVE_HOME);
+                                    jules->RemoveAura(SPELL_JULES_VOMITS_AURA);
+                                }
+
+                                events.ScheduleEvent(EVENT_BARADAS_TALK, 10000);
+                                step++;
+                                break;
+                            case 21:
+                                //End
+                                if (playerGUID)
+                                {
+                                    if (Player* player = ObjectAccessor::FindPlayer(playerGUID))
+                                        player->KilledMonsterCredit(NPC_COLONEL_JULES, 0);
```

Comment:
/home/travis/build/TrinityCore/TrinityCore/src/server/scripts/Outland/zone_hellfire_peninsula.cpp:712:88: fatal error: 
      no viable conversion from 'int' to 'ObjectGuid'
  ...player->KilledMonsterCredit(NPC_COLONEL_JULES, 0);

Try ObjectGuid::Empty instead of 0